spyder/plugins/projects/plugin.py

```diff
@@ -203,6 +206,16 @@ def on_main_menu_available(self):
             menu_id=ApplicationMenus.Projects,
             section=ProjectsMenuSections.Extras)
 
+    @on_plugin_available(plugin=Plugins.Switcher)
+    def on_switcher_available(self):
+        # Connect to switcher
+        self._switcher = self.main.switcher
```

Comment:
```suggestion
        self._switcher = self.get_plugin(Plugins.Switcher)
```

We usually prefer to get references to other plugins by using the `get_plugin` method.